exercises/robot-simulator/canonical-data.json

```diff
@@ -31,7 +33,8 @@
           }
         },
         {
-          "description": "Negative positions are allowed",
+          "description": "Allows negative positions",
+          "comments": [ "Negative positions allowed" ],
```

Comment:
The comment just repeats what the description says, so it could be removed. Same with the comments in L58, L62, L82 etc

Reply:
I thought the same when I did it.  I was just trying to not to lose any info.  I'll wait to see how the other discussion pans out be doing such cleanup.